sites/shahid.mbc.net/shahid.mbc.net.config.js

```diff
@@ -34,24 +34,28 @@ module.exports = {
   async channels({ lang = 'en' }) {
```

Comment:
A best practice in my opinion would have been to change the parameters that "channels" get in order to inject the country codes of our choice.


```suggestion
      async channels({ lang = 'en', country `})`
```

This way, you can do 

```
npm run channels:parse --- --config=./sites/shahid.mbc.net/shahid.mbc.net.config.js --output=./sites/shahid.mbc.net/shahid.mbc.net_eg-en.channels.xml --set=country:eg --set=lang:en
````